prometheus/common/auto.d/ProductionDB.yml.example

```diff
@@ -8,3 +8,9 @@
   labels:
     job: "Prod"
     exp_type: "pg"
+    cluster: "crunchy"
+- targets: [ "10.123.202.12:9188" ]
+  labels:
+    job: "Prod"
```

Comment:
Add a note to use cluster_node naming ?

Reply:
I updated this variable to be `cluster_name`. Not sure what sort of note you mean to add.

Reply:
job: "Prod" or job: "mycluster_Prod"

Reply:
> job: "Prod" or job: "mycluster_Prod"

With the `cluster_name` tag, there's really no need for that anymore

Reply:
I'm fine either way though if you have a preference

Reply:
Updated to include the cluster_name in the job name